### PR TITLE
MNT Use lint and black format

### PR DESCRIPTION
**Pull request by ArturoAmorQ**
This may create conflicts with previously opened PRs. Please merge this PR before any other.

### CODE REVIEW

File: python_scripts/02_numerical_pipeline_introduction.py [hunk 1]
```diff
@@ -160,71 +160,73 @@
...
 # as in the beginning of this notebook.
 
 # %%
 target_test = adult_census_test[target_name]
-data_test = adult_census_test.drop(columns=[target_name, ])
+data_test = adult_census_test.drop(
+    columns=[
+        target_name,
+    ]
+)
```
Comment (ArturoAmorQ) [inline python_scripts/02_numerical_pipeline_introduction.py:195]:
```suggestion
data_test = adult_census_test.drop(columns=[target_name])
```

File: python_scripts/cross_validation_stratification.py [hunk 2]
```diff
@@ -67,13 +67,12 @@
 cv = KFold(n_splits=3)
 results = cross_validate(model, data, target, cv=cv)
 test_score = results["test_score"]
-print(f"The average accuracy is "
-      f"{test_score.mean():.3f} ± {test_score.std():.3f}")
+print(f"The average accuracy is " f"{test_score.mean():.3f} ± {test_score.std():.3f}")
```
Comment (ArturoAmorQ) [inline python_scripts/cross_validation_stratification.py:70]:
```suggestion
print(f"The average accuracy is {test_score.mean():.3f} ± {test_score.std():.3f}")
```

**Comment by ogrisel** [comment]
Did you use the default line length of 88 columns or a custom one?

We probably need a pre-commit config file and a CI entry to run the linter.

In `hazardous` I configured the pre-commit as follows:

- https://github.com/soda-inria/hazardous/blob/main/.pre-commit-config.yaml

and then reused it in the linter config for the CI:

- https://github.com/soda-inria/hazardous/blob/2b2cbd5fe6696c225f0c9dccae0c9dcfd0cc1eeb/.github/workflows/test.yml#L13-L27

**Comment by ogrisel** [comment]
we also need to configure the black line length in `pyproject.toml`.

**Comment by ogrisel** [comment]
Maybe we can try:

```
[tool.nbqa]
ignore = ["# noqa", "%%"]
```

from https://you.com/search?q=make++black+ignore+%25%25+cell+magic&tbm=youchat&fromExtension=true&cid=c0_d133d0e8-aeab-457b-a263-e4c292233ef1

**Review by glemaitre — APPROVED** [review]

**Review by ogrisel — APPROVED** [review]

**Pull request merged**